files/get_from_bucket.sh

```diff
@@ -0,0 +1,60 @@
+#! /bin/bash
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Given a url and filename, download an object to the vardir. This function uses
+# gsutil to fetch from the bucket.  Note, the service account for the instance
+# must be properly configured with a role having authorization to get objects
+# from the bucket.  See Product/mktmail/commit/39fa9b8 for an example of the
```

Comment:
What is `Product/mktmail/commit/39fa9b8` ?

Reply:
Cruft, removed it in 513dc7b.  Good catch.